jas/common.go

```diff
@@ -270,6 +270,25 @@ func convertToFilesExcludePatterns(excludePatterns []string) []string {
 	return patterns
 }
 
+func CheckForSecretValidation(xrayManager *xray.XrayServicesManager, validateSecrets bool) bool {
+	// Ordered By importance
+	// first check for flag
+	if validateSecrets {
+		return validateSecrets
+	}
+	// second check for env var
+	if strings.ToLower(os.Getenv("JF_VALIDATE_SECRETS")) == "true" {
+		return true
+	}
```

Comment:
```suggestion
	// Ordered By importance: flag, env var and value stored remote (platform API)
    if validateSecrets || strings.ToLower(os.Getenv(JfSecretValidationEnvVariable)) == "true" {
```
Use the consts, you can also merge some conditions